### PR TITLE
List of items with an icon on the left

### PR DESCRIPTION
**Pull request by mdeniz**

### CODE REVIEW

**Comment by hennevogel** [comment]
Deploy preview for *obs-patterns* ready!

Built with commit 7f532061d3d2dd32fc4c637afcd576a7384b2906

https://deploy-preview-6--obs-patterns.netlify.com

File: _posts/components/2018-4-1-list-with-icon.html [hunk 1]
```diff
@@ -0,0 +1,73 @@
+---
+layout: post
+title:  "List of items"
+category: components
+description: For having a list of items with an icon in the left side.
```
Comment (dmarcoux) [inline _posts/components/2018-4-1-list-with-icon.html]: `on the left side`

File: _posts/components/2018-4-1-list-with-icon.html [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+             If this is integrated in body of a card, then add "list-group-item-integrated.border-left-0.border-right-0"
```
Comment (dmarcoux) [inline _posts/components/2018-4-1-list-with-icon.html]: `list-group-item-integrated.border-left-0.border-right-0`

File: _posts/components/2018-4-1-list-with-icon.html [hunk 3]
```diff
@@ -0,0 +1,73 @@
+---
+layout: post
+title:  "List of items"
+category: components
+description: For having a list of items with an icon in the left side.
+             If this is integrated in body of a card, then add "list-group-item-integrated.border-left-0.border-right-0" 
+             classes for removing duplicate borders to the "list-group-item" div
```
Comment (dmarcoux) [inline _posts/components/2018-4-1-list-with-icon.html]: of *divs* with the class `list-group-item`

**Review by dmarcoux — CHANGES_REQUESTED** [review]
Nice work.

File: _posts/components/2018-4-1-list-with-icon.html [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+---
+
+%p
```
Comment (dmarcoux) [inline _posts/components/2018-4-1-list-with-icon.html]: Are these 2 `<p>` needed?

File: _posts/components/2018-4-1-list-with-icon.html [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+      .list-group-item
+        .row
+          .col-2.pl-2.text-center
+            %i.fa.fa-code-branch.fa-2x.text-success.pt-2{title: 'Icon title 3'}
+          .col-10.pl-0
+            %a{href: "http://example.com"} Example Link 3
+            %p.small.mb-0
+              Second line of text for item 3
+%p
+%p
```
Comment (dmarcoux) [inline _posts/components/2018-4-1-list-with-icon.html:41]: Again, are these 2 `<p>` needed?

File: _posts/components/2018-4-1-list-with-icon.html (hunk 4 above)
Comment (mdeniz) [reply _posts/components/2018-4-1-list-with-icon.html]: Are needed to separate the visual examples displayed. Aren't needed for using the code example... the main problem is that the code you add is used for showing the visual part and to show the code

File: _posts/components/2018-4-1-list-with-icon.html (hunk 5 above)
Comment (mdeniz) [reply _posts/components/2018-4-1-list-with-icon.html:41]: the same as before

File: _posts/components/2018-4-1-list-with-icon.html (hunk 3 above)
Comment (mdeniz) [reply _posts/components/2018-4-1-list-with-icon.html]: do you prefer "elements"

File: _posts/components/2018-4-1-list-with-icon.html (hunk 5 above)
Comment (hennevogel) [reply _posts/components/2018-4-1-list-with-icon.html:41]: Let's solve this with CSS for the site and not with code in the example please...

Comment (dmarcoux) [reply _posts/components/2018-4-1-list-with-icon.html:41]: We already have a format for examples, so we can change it and then adapt the few examples we have now to fix this.

Comment (dmarcoux) [reply _posts/components/2018-4-1-list-with-icon.html:41]: I discussed with @mdeniz, I will take care of it in another PR.

**Review by dmarcoux — APPROVED** [review]

**Pull request merged**